lib/msal-browser/src/utils/CryptoConstants.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Cryptographic Algorithms used/supported
+export enum Algorithms  {
+    PKCS1_V15_KEYGEN_ALG = "RSASSA-PKCS1-v1_5",
+    RSA_OAEP = "RSA-OAEP",
+    AES_GCM = "AES-GCM",
+    DIRECT = "dir",
+    S256_HASH_ALG = "SHA-256",
+}
+
+// Numerical constants relating to bit/byte length
+export enum Lengths {
+    modulus = 2048
+}
+
+// Public Exponent used in Key Generation
+export const PUBLIC_EXPONENT = new Uint8Array([0x01, 0x00, 0x01]);
+
+/**
+ * JWK Key Format string (Type MUST be defined for window crypto APIs)
+ */
+export enum CryptoKeyFormats  {
+    jwk = "jwk"
+}
+
+// Crypto Key Usage sets
+export const KEY_USAGES = {
```

Comment:
A lot of inconsistency in terms of casing here. Can these be capital case?

```suggestion
export const KeyUsages = {
```

Reply:
I went with the all caps snake case for consts and pascal case for enums, but I think making them all pascal case would look better.